src/util-ebpf.c

```diff
@@ -83,6 +87,15 @@ static void BpfMapsInfoFree(void *bpf)
     int i;
     for (i = 0; i < bpfinfo->last; i ++) {
         if (bpfinfo->array[i].name) {
+            if (bpfinfo->array[i].unlink) {
+                char pinnedpath[1024];
+                snprintf(pinnedpath, sizeof(pinnedpath),
```

Comment:
check snprintf before passing it to unlink

Reply:
indeed